src/canisters/governance/response.converters.ts

```diff
@@ -198,17 +194,10 @@ const toProposal = ({
 const toAction = (action: RawAction): Action => {
   if ("ExecuteNnsFunction" in action) {
```

Comment:
will this function still works out afterwards - i.e. will `MakeProposal` that uses it still works out?

Reply:
This `toAction` is used only in response conversion, so it should not break any `NOT` FE related functionality. To create a dummy proposal with nnsFunction another function is used `makeExecuteNnsFunctionDummyProposalRequest`.

Reply:
Gotcha, thanks for the feedback